### DISCUSSION TITLE
Integrated Transcription - only lower case characters

### DISCUSSION THREAD

**Discussion by XanomalieX** [SWivid/F5-TTS#512] [Q&A]
Hi I just realized, that the integrated transcription (seems to be faster-whisper) only outputs lower case characters.
Is that bad for european languages, or irrelevant, since the cleaner in the inference changes everything higher case charcters?

**Comment by SWivid** [comment]
No specific text normalization yet, no capitalization.
In the lastest version, we have included features that the transcription result will be automatically filling into the ref_text box, you could modify based on that, which is more convenient.
